src/main/target/REVO/target.h

```diff
@@ -69,10 +73,6 @@
 #define BEEPER                  NONE
 #endif
 
-#if defined(REVOLT)
```

Comment:
It looks as though this was accidentally removed. Same with the block 153..158. Try soft reset to betaflight/master and `git add -p`/`git checkout -p betaflight/master .` to only add what you need